rest-api/offline/table_exporter.py

```diff
@@ -45,10 +45,10 @@ def _obfuscate_id(pmi_id, salt):
     h = hashlib.sha1()
     h.update(struct.pack('>l', pmi_id))
     h.update(salt)
-    # Just take the first 4 bytes so that the output ID is an integer, roughly in the same domain as
+    # Just take the first 8 bytes so that the output ID is a long, roughly in the same domain as
     # the input PMI participant ID.
-    b = h.digest()[0:4]
-    return abs(struct.unpack('>l', b)[0])
+    b = h.digest()[0:8]
+    return abs(struct.unpack('>q', b)[0])
```

Comment:
What's >q for? Add a comment?

Reply:
It's a format string for the struct package, added a comment.